backend/dissemination/models.py

```diff
@@ -126,7 +126,7 @@ class FederalAward(models.Model):
 
     award_seq_number = models.IntegerField(
         "Order that the award line was reported",
-        null=True,
+        null=False,
```

Comment:
You could just delete this line.  It will default to `null=False` if not specified.  But I'm fine with it either way.